manifests/importer.pp

```diff
@@ -12,7 +12,7 @@
     client_enabled => true,
   }
   Nfs::Client::Mount <<| nfstag == 'rsan' |>>
-
+    { options_nfs => 'tcp,nolock,rsize=32768,wsize=32768,soft,noatime,actimeo=3,retrans=1',}
```

Comment:
Can this just be added to the exported resource rather than using a resource collector override to append later?

Resource collector overrides are disgusting, but sometimes just the think you need, like a bloody mary with anchovies. But I'm not sure this is a bloody mary with anchovies situation 🍹

Reply:
ill give it a shot, there is not reason why not I guess,

Reply:
Yeah because if it was left as it is, it means that if later on you were to add `options_nfs` to the resource itself, it would have no effect since it's being overridden here, which would take a long time to debug I reckon

Reply:
@dylanratcliffe should be completed